hoptimator-jdbc/src/main/java/com/linkedin/hoptimator/jdbc/HoptimatorDdlExecutor.java

```diff
@@ -187,7 +187,12 @@ public void execute(SqlCreateMaterializedView create, CalcitePrepare.Context con
 
       // Support "partial views", i.e. CREATE VIEW FOO$BAR, where the view name
       // is "foo-bar" and the sink is just FOO.
-      String sinkName = viewName.split("\\$", 2)[0];
+      String[] viewParts = viewName.split("\\$", 2);
+      String sinkName = viewParts[0];
+      String partialViewName = "";
```

Comment:
We should use `null` instead of empty string.

Reply:
I liked the idea of pipelineName, way cleaner and now don't have to worry about nulls